projects/addon-doc/package.json

```diff
@@ -16,7 +16,7 @@
     "dependencies": {
         "@angular-devkit/schematics": "12.2.18",
         "markdown-it": "13.0.1",
-        "ngx-highlightjs": "6.1.3",
+        "ngx-highlightjs": "5.0.0",
```

Comment:
The downgrading will cause [the fail](https://github.com/taiga-family/taiga-ui/actions/runs/5854268319/job/15869857358?pr=5097) of this test
https://github.com/taiga-family/taiga-ui/blob/9bd822136cf2b1cf62d9ab513774a8bfea2a4260/projects/addon-doc/components/code/tests/highlight.spec.ts#L121
<img width="1277" alt="failed-test" src="https://github.com/taiga-family/taiga-ui/assets/35179038/377aea49-1d89-4bdb-aef7-f5662fbf94e5">


**Why?**
It is bug of `ngx-highlightjs`. It was fixed in [6.1.1](https://github.com/MurhafSousli/ngx-highlightjs/releases/tag/v6.1.1).
<img width="947" alt="611" src="https://github.com/taiga-family/taiga-ui/assets/35179038/e6fb0fad-22e2-4987-83c6-401ae76c1de1">